src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.4.0.0.cs

```diff
@@ -47,11 +47,25 @@ public static void DefaultSettings_Http_Text_Echo_RoundTrips_String()
     }
 
     // Https: Client and Server bindings setup exactly the same using default settings.
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
     [ConditionalFact(nameof(Root_Certificate_Installed))]
-    [OuterLoop]
     [ActiveIssue(1123, PlatformID.AnyUnix)]
+#endif
+    [OuterLoop]
     public static void DefaultSettings_Https_Text_Echo_RoundTrips_String()
     {
+#if FULLXUNIT_NOTSUPPORTED
+        bool root_Certificate_Installed = Root_Certificate_Installed();
+        if (!root_Certificate_Installed)
+        {
+            Console.WriteLine("---- Test SKIPPED --------------");
+            Console.WriteLine("Attempting to run the test in ToF, a ConditionalFact evaluated as FALSE.");
```

Comment:
Non-blocking nit from pre-PR review -- I'd prefer to combine these into a single line, but agree it's okay to merge as is.